### ISSUE TITLE
Update XML namespaces for JSF 2.2+

### ISSUE THREAD

**Issue by poikilotherm** [IQSS/dataverse#6363]
Since JSF 2.2, the XML namespaces URLs have changed - the old Sun based are still working, but not updated with newer elements like `f:viewAction` etc.

For validation, autocompletion etc. this should be fixed in `*.xhtml` under `/src/main/webapp`.

```diff
-      xmlns:h="http://java.sun.com/jsf/html"
-      xmlns:f="http://java.sun.com/jsf/core"
-      xmlns:ui="http://java.sun.com/jsf/facelets"
+      xmlns:h="http://xmlns.jcp.org/jsf/html"
+      xmlns:f="http://xmlns.jcp.org/jsf/core"
+      xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
```

See also: https://stackoverflow.com/questions/31068678/which-xml-namespace-to-use-with-jsf-2-2-and-up

I could not find anything wether this namespace URLs are going to change due to the transition from Java EE to Jakarta EE. It's very likely that this will either stay or stay backward compatible.

**Comment by pdurbin** [comment]
At https://jakarta.ee/specifications/faces/2.3/vdldoc/overview-summary.html I'm also seeing jcp.org stuff, like this:

    xmlns:h="http://xmlns.jcp.org/jsf/html

**Comment by poikilotherm** [comment]
I guess this will be addressed with the moved for JSF 4.0 in #8305.

Closing this one.
